packages/app-tests/e2e/pr-718-add-stepper-component.spec.ts

```diff
@@ -71,5 +64,5 @@ test(`[PR-718]: should be able to create a document`, async ({ page }) => {
   await page.waitForURL('/documents');
 
   // Assert document was created
-  await expect(page.getByRole('link', { name: documentTitle })).toBeVisible();
+  await page.getByRole('link', { name: documentTitle }).click();
```

Comment:
Successfully generate a document so it can be removed when the user is deleted after the test finishes